feature_engine/timeseries/forecasting/base_forecast_transformers.py

```diff
@@ -61,10 +66,17 @@ class BaseForecastTransformer(BaseEstimator, TransformerMixin, GetFeatureNamesOu
     def __init__(
         self,
         variables: Union[None, int, str, List[Union[str, int]]] = None,
+        fill_value: Union[float, int, str] = None,
```

Comment:
I think the default value should be that of pandas shift:

https://github.com/pandas-dev/pandas/blob/91111fd99898d9dcaa6bf6bedb662db4108da6e6/pandas/core/frame.py#L5766